.github/workflows/rosa-stretched-cluster.yml

```diff
@@ -0,0 +1,189 @@
+        default: "a,b,c"
+
+jobs:
+  meta:
+    name: Metadata
+    runs-on: ubuntu-latest
+    outputs:
+      cidr: ${{ steps.cluster.outputs.CIDR }}
+      clusterName: ${{ steps.cluster.outputs.CLUSTER_NAME }}
+      region: ${{ steps.cluster.outputs.REGION }}
+      availabilityZones: ${{ steps.cluster.outputs.ZONES }}
+      numberOfZones: ${{ steps.cluster.outputs.NUMBER_ZONES }}
+    concurrency:
+      group: multi-az-meta
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup ROSA CLI
+        uses: ./.github/actions/rosa-cli-setup
+        with:
+          aws-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
+          rosa-token: ${{ secrets.ROSA_TOKEN }}
+
+      - name: Compute Cluster Info
+        id: cluster
+        working-directory: provision/aws
+        run: |
+          CLUSTER_NAME=${{ inputs.clusterName || format('gh-{0}-az', github.repository_owner) }}
+          REGION=${{ inputs.region || vars.AWS_DEFAULT_REGION }}
+          ZONES=$(./prefix_region_to_zones.sh "${REGION}" "${{ inputs.availabilityZones }}")
+          CIDR=$(CLUSTER_PREFIX=${CLUSTER_NAME} ./rosa_machine_cidr.sh | jq -r .cidr_a)
+          IFS=, read -r -a AZS_ARRAY <<< "${ZONES}"
+          NUMBER_ZONES="${#AZS_ARRAY[@]}"
+
+          echo "CLUSTER_NAME=${CLUSTER_NAME}" >> ${GITHUB_OUTPUT}
+          echo "REGION=${REGION}" >> ${GITHUB_OUTPUT}
+          echo "ZONES=${ZONES}" >> ${GITHUB_OUTPUT}
+          echo "CIDR=${CIDR}" >> ${GITHUB_OUTPUT}
+          echo "NUMBER_ZONES=${NUMBER_ZONES}" >> ${GITHUB_OUTPUT}
+
+  cluster:
+    name: Create Openshift Cluster
+    if: ${{ inputs.createCluster }}
+    needs: meta
+    uses: ./.github/workflows/rosa-cluster-create.yml
+    with:
+      cidr: ${{ needs.meta.outputs.cidr }}
+      clusterName: ${{ needs.meta.outputs.clusterName }}
+      region: ${{ needs.meta.outputs.region }}
+      availabilityZones: ${{ needs.meta.outputs.availabilityZones }}
+      useSingleAz: false
+    secrets: inherit
+
+  monitoring:
+    name: Install Grafana
+    if: ${{ inputs.createCluster }}
+    runs-on: ubuntu-latest
+    needs:
+      - meta
+      - cluster
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Go Task
+        uses: ./.github/actions/task-setup
+
+      - name: Setup ROSA CLI
+        uses: ./.github/actions/rosa-cli-setup
+        with:
+          aws-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
+          rosa-token: ${{ secrets.ROSA_TOKEN }}
+
+      - name: Login to OpenShift cluster
+        uses: ./.github/actions/oc-keycloak-login
+        with:
+          clusterName: ${{ needs.meta.outputs.clusterName }}
+
+      - name: Deploy Grafana
```

Comment:
The job and task are called "monitoring", but here and the job .name refer to Grafana. We should be consistent and just use "monitoring" everywhere like we do with the xsite clusters.